src/eval.c

```diff
@@ -392,7 +392,7 @@ sds luaCreateFunction(client *c, robj *body) {
     if (luaL_loadbuffer(lctx.lua,funcdef,sdslen(funcdef),"@user_script")) {
         if (c != NULL) {
             addReplyErrorFormat(c,
-                "Error compiling script (new function): %s\n",
+                "Error compiling script (new function): %s",
```

Comment:
@MeirShpilraien why are the newlines here? (and in the other error below)

Reply:
Here its not needed, above I guess you mean here: https://github.com/redis/redis/pull/10279/files#diff-3f6891fec05432f415cfd49150db67e2bc72d20e0a94a4e3ac762f7ff55f9921R250 ?
If so its a script to you need to specify end of line.

Reply:
i mean that i think you should mirror these 3 changes (trimming excessive newline in 3 places) to your PR (the one that replaces this one)

Reply:
Right, will add it.